src/Executables/PrintMinimumGridSpacing/PrintMinimumGridSpacing.hpp

```diff
@@ -72,11 +74,15 @@ struct InitializeElement {
         make_not_null(&box), [&domain](const auto domain_ptr) noexcept {
           domain = std::move(*domain_ptr);
         });
+
     return std::make_tuple(
-        elliptic::Initialization::Domain<Dim>::initialize(
-            db::create_from<typename AddOptionsToDataBox::simple_tags>(
-                std::move(box)),
-            array_index, initial_extents, domain),
+        db::create_from<db::RemoveTags<>,
+                        db::AddSimpleTags<>,
+                        compute_tags>(
+            elliptic::Initialization::Domain<Dim>::initialize(
+                db::create_from<typename AddOptionsToDataBox::simple_tags>(
+                    std::move(box)),
+                array_index, initial_extents, domain)),
         true);
```

Comment:
@geoffrey4444 I was able to incorporate the `MinimumGridSpacing` `ComputeTag` from within the `PrintMinimumGridSpacing` executable, leaving `src/Elliptic/Initialization/Domain.hpp` unmodified, but I think there may be a cleaner way of incorporating it than a nested call to `create_from`. I'm referring to this specific edit to the above `return`.

Reply:
I don't know of a better way, but other spectre reviewers might. For now, if it works, let's leave this

Reply:
There's an `AddComputeTags` action in #1466.  I haven't looked to see whether it would actually be useful here, (I'm just commenting based on the name.) and of course it isn't in develop yet.

Reply:
With the `AddComputeTags` action that @wthrowe mentioned and the domain initializer I'm adding in #1612 you should be able to get rid of this `InitializeElement` action entirely. You can look at one of the evolution executables for an example. If those two PRs get merged before this one, then I suggest you use that initialization pattern.